tests/extendrtests/tests/testthat/test-wrappers.R

```diff
@@ -1,3 +1,31 @@
+test_that("Wrapper code is up-to-date", {
+  # if this test fails check whether the wrapper code needs 
+  # updating via make_wrappers()
+  x <- .Call(
+    "wrap__make_extendrtests_wrappers",
+    use_symbols = TRUE,
+    package_name = "extendrtests"
+  )
+  x <- strsplit(x, "\n")[[1]]
+  
+  # testthat run locally?
+  tmp <- file.path("..", "..", "R", "extendr-wrappers.R")
+  if (file.exists(tmp)) {
+    source <- tmp
+  } else {
+    # testthat run as part of R CMD check
+    tmp <- file.path("..", "..", "00_pkg_src", "extendrtests", "R", "extendr-wrappers.R")
+    if (file.exists(tmp)) {
+      source <- tmp
+    } else {
+      skip("Cannot locate wrapper code.")
+    }
+  }
+  
+  y <- brio::read_lines(source)
+  expect_equal(x, y)
```

Comment:
Perhaps add a few sentences atop of `test_that` to explain what is achieved here (again, it was not immediately evident to me what is tested), as the test body looks complicated.
Something like `"check the existence of 'extendr-wrappers.R' and compare it to the generated up-to-date version. If this test fails...."`. Use better wording, I am not good at this :)

Reply:
Done.